src/Bicep.Core/Configuration/bicepconfig.json

```diff
@@ -1,4 +1,25 @@
 {
+  "cloud": {
+    "currentProfile": "AzureCloud",
+    "profiles": {
+      "AzureCloud": {
+        "resourceManagerEndpoint": "https://management.azure.com"
+      },
+      "AzureChinaCloud": {
+        "resourceManagerEndpoint": "https://management.chinacloudapi.cn"
+      },
+      "AzureUSGovernment": {
+        "resourceManagerEndpoint": "https://management.usgovcloudapi.net"
+      },
+      "Dogfood": {
+        "resourceManagerEndpoint": "https://api-dogfood.resources.windows-int.net"
+      }

```

Comment:
Do we want to expose this?

Reply:
Yeah good pt. I think we should exclude this one as well and use the custom endpoint as a way of testing with that cloud.

Reply:
Technically this is invisible to users, and I added it because Azure CLI exposes it in `az cloud list`, but am totally open to remove it if there's concern.

Reply:
Removed.